src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/SemanticTokenTestBase.cs

```diff
@@ -22,7 +23,6 @@
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 using Xunit;
 using Xunit.Sdk;
-using OS = OmniSharp.Extensions.LanguageServer.Protocol.Models;
 using VS = Microsoft.VisualStudio.LanguageServer.Protocol;
```

Comment:
This seems unnecessary, as the namespace is already imported, so there should be no need to disambiguate, right?

Reply:
Range still conflicts with some System namespaces, but I can target this to just alias MS.VS.LS.Range instead.